docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -328,20 +333,23 @@ contract MyPaymaster is IPaymaster {
             (bool success, ) = payable(BOOTLOADER_FORMAL_ADDRESS).call{
                 value: requiredETH
             }("");
-            require(success, "Failed to transfer funds to the bootloader");
+            require(
+                success,
+                "Failed to transfer tx fee to the bootloader. Paymaster balance might not be enough."
+            );
         } else {
             revert("Unsupported paymaster flow");
         }
     }
 
-    function postTransaction  (
+    function postTransaction(
         bytes calldata _context,
         Transaction calldata _transaction,
         bytes32,
         bytes32,
         ExecutionResult _txResult,
         uint256 _maxRefundedGas
-    ) onlyBootloader external payable override {
```

Comment:
?

Reply:
fix incoming

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/465/commits/3ae8c68a33e059a1bf7fdf6f4a84ae2185722510